packages/next-app/src/components/Tabs.tsx

```diff
@@ -0,0 +1,81 @@
+import * as TabsPrimitive from "@radix-ui/react-tabs";
+import cn from "classnames";
+import { createContext, useContext } from "react";
+
+enum Colors {
+  BLUE = "blue7",
+  AMBER = "amber9",
+}
+
```

Comment:
Those colors were chosen because of the variant and baseline colors on the plots. This means that this shouldn't be used for the Tokens tab.

Reply:
not quite sure if a follow, we are currently using these colors for an underline when that tab is active 
<img width="347" alt="image" src="https://github.com/bleu-studio/balancer-pool-metadata/assets/63298341/24fca0bb-acb9-44b0-822b-2bab117d5394">

If this shouldn't be used for the Tokens tab, should we pass the color directly as a prop to the tab component?

Reply:
IMO it is an optional parameter. If we don't pass consider white to all tabs.

Reply:
idk why the underline is not working with dynamic values (depending on the color). So I'm letting be always white for now as this is not so valuable

Reply:
not sure if related, but remember dynamic class values are not supported by tailwind https://tailwindcss.com/docs/content-configuration#dynamic-class-names

Reply:
😢